index.js

```diff
@@ -800,10 +886,17 @@ module.exports = function(THREE) {
 
         }
 
-        function touchend( /* event */ ) {
+        function touchend( event ) {
 
             if ( scope.enabled === false ) return;
 
+            if ( isDoubleClick( event ) === true ) {
```

Comment:
I believe the doubletap can be initiated only after the touch state has been completed - after  state = STATE.NONE;